.github/workflows/run-tests.yml

```diff
@@ -42,13 +47,22 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Cache chianti database
+      uses: actions/cache@v2
+      if: matrix.toxenv != 'build_docs' && matrix.toxenv != 'codestyle'
+      env:
+        # Bump the version number when changing the chianti version to test with
+        cache-name: cache-chianti-dbase-807
+      with:
+        path: ~/.chianti
+        key: ${{ env.cache-name }}-${{ matrix.toxenv }}
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: python -m pip install --upgrade tox codecov
     - name: Run tox
-      run: python -m tox -e  ${{ matrix.toxenv }}
+      run: python -m tox -e  ${{ matrix.toxenv }} ${{ matrix.cache_cmd }}
```

Comment:
This has just added the cache_cmd environment variable to be passed to the tox command, which sets where the database is downloaded to.